src/renderer/components/depositActions/forms/Leave.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback } from 'react'
+
+import { Form } from 'antd'
+import { useIntl } from 'react-intl'
+
+import { Input } from '../../uielements/input'
+import * as Styled from './Forms.styles'
+
+const getLeavedMemo = (thorAddress: string) => `LEAVE:${thorAddress}`
```

Comment:
^ Let provide a LEAVE memo helper by `asgardex-util` (with another PR if you want) - new issue :eyes:  https://gitlab.com/thorchain/asgardex-common/asgardex-util/-/issues/17